docs/faq/questions/cloud-faq.mdx

```diff
@@ -56,6 +56,22 @@ Cloud.
 A [user](/guides/cloud/users) is anyone with a login to Cypress Cloud who has
 been invited to see and review the test results of your organization.
 
+## <Icon name="angle-right" /> What if I can't access my Cypress Cloud account?
+
```

Comment:
Weird, I posted this update a few days ago. Maybe I never saved it. My bad on this suggestion I made earlier. I didn't notice the line break with the word "instructions" on the next line. If you pull out the "." I suggested after the link? Then we should be in good shape.